src/plugins/faultManagement/FaultManagementListView.vue

```diff
@@ -0,0 +1,333 @@
+<template>
+<div class="c-faults-list-view">
+    <FaultManagementSearch
+        :searchTerm="searchTerm"
+        @filterChanged="updateFilter"
+        @updateSearchTerm="updateSearchTerm"
+    />
+
+    <FaultManagementToolbar
+        :selected-faults="selectedFaults"
+        @acknowledgeSelected="toggleAcknowledgeSelected"
+        @shelveSelected="toggleShelveSelected"
+    />
+
+    <FaultManagementListHeader
+        class="header"
+        :selected-faults="Object.values(selectedFaults)"
+        :total-faults-count="filteredFaultsList.length"
+        @selectAll="selectAll"
+        @sortChanged="sortChanged"
+    />
+
+    <template v-if="filteredFaultsList.length > 0">
+        <FaultManagementListItem
+            v-for="fault of filteredFaultsList"
+            :key="fault.key"
+            :fault="fault"
+            :is-selected="isSelected(fault)"
+            @toggleSelected="toggleSelected"
+            @acknowledgeSelected="toggleAcknowledgeSelected"
+            @shelveSelected="toggleShelveSelected"
+        />
+    </template>
+</div>
+</template>
+
+<script>
+import FaultManagementListHeader from './FaultManagementListHeader.vue';
+import FaultManagementListItem from './FaultManagementListItem.vue';
+import FaultManagementSearch from './FaultManagementSearch.vue';
+import FaultManagementToolbar from './FaultManagementToolbar.vue';
+
+import { FAULT_MANAGEMENT_SHELVE_DURATIONS_IN_MS, FILTER_ITEMS, SORT_ITEMS } from './constants';
+
+export default {
+    components: {
+        FaultManagementListHeader,
+        FaultManagementListItem,
+        FaultManagementSearch,
+        FaultManagementToolbar
+    },
+    inject: ['openmct', 'domainObject'],
+    props: {
+        faultsList: {
+            type: Array,
+            default: () => []
+        }
+    },
+    data() {
+        return {
+            filterIndex: 0,
+            searchTerm: '',
+            selectedFaults: {},
+            sortBy: Object.values(SORT_ITEMS)[0].value
+        };
+    },
+    computed: {
+        filteredFaultsList() {
+            const filterName = FILTER_ITEMS[this.filterIndex];
+            let list = this.faultsList.filter(fault => !fault.shelveInfo);
+            if (filterName === 'Acknowledged') {
+                list = this.faultsList.filter(fault => fault.acknowledged);
+            }
+
+            if (filterName === 'Unacknowledged') {
+                list = this.faultsList.filter(fault => !fault.acknowledged);
+            }
+
+            if (filterName === 'Shelved') {
+                list = this.faultsList.filter(fault => fault.shelveInfo);
+            }
+
+            if (this.searchTerm.length > 0) {
+                list = list.filter(this.filterUsingSearchTerm);
+            }
+
+            list.sort(SORT_ITEMS[this.sortBy].sortFunction);
+
+            return list;
+        }
+    },
+    watch: {},
```

Comment:
Unused, let's remove. Is the linter complaining if we don't have these or something?